oat-sa/literallycanvas
# fix: remove vulnerabilities

Ticket: https://oat-sa.atlassian.net/browse/ADF-1382

## What's Changed

- Removed vulnerabilities

## How to test
- Clone the repo and run `yarn audit`

## How to test on env
- go to http://test-viktar.playground.kitchen.it.taocloud.org:41531/
- login as a guest
- click start button
- click tools dropdown on the bottom-left corner
- choose Scratchpad
- the canvas will appear so you can test it

![image](https://github.com/oat-sa/literallycanvas/assets/135347298/7e9276a2-f82c-4bbb-b869-eb56e11b0b63)
